auth-web/src/store/modules/business.ts

```diff
@@ -51,10 +52,19 @@ export default class BusinessModule extends VuexModule {
       (nr.state === NrState.CONDITIONAL &&
         [NrConditionalStates.RECEIVED, NrConditionalStates.WAIVED].includes(nr.consentFlag))
 
-    /** Returns True if NR is approved for one stop registration. */
+    /** Returns True if NR is approved for registration. */
     const isApprovedForRegistration = (nr): boolean =>
       nr.actions.some(action => action.filingName === LearFilingTypes.REGISTRATION)
 
+    /** Returns target conditionally. */
+    const getTarget = (nr): NrTargetTypes => {
+      if ([LegalTypes.SP, LegalTypes.GP].includes(nr.legalType) && !enableSpGpDba) {
+        // Fallback to onestop if SpGpDba is not enabled
+        return NrTargetTypes.ONESTOP
+      }
+      return nr.target
+    }
+
```

Comment:
Isn't the target be handled by the API? 
Or is this just a temporary means of feature flagging our redirect?

Reply:
It is handled in API. It is temporary until we migrate to lear

Reply:
Gotcha, thanks!